models/_sector/nft/trades/solana/nft_solana_trades.sql

```diff
@@ -0,0 +1,72 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        , materialized = 'view'
```

Comment:
we can explore materializing this, similar to how we do `nft.trades`
unless the ultimate plan is to keep this as a view, then add to `nft.trades` where it's materialized